### DISCUSSION TITLE
share link to the track (or album)

### DISCUSSION THREAD

**Discussion by MG-Sky** [navidrome/navidrome#2378] [Ideas]
Im wonder if this is possible to have an option to share a link to the track or album - 
fe. i ve logged and now im listening to the nice track so I want to share this track with someone who maybe has no account, so maybe an option "share with link..." would work here and allow me to copy url to the track with an token that someone who has no account can listen (maybe even for some period of the time).

**Comment by deluan** [comment]
This is a feature that you need to enable in your configuration as it is still considered experimental. I just put together a page explaining the feature and how to enable it: https://www.navidrome.org/docs/usage/sharing/

**Comment by awsms** [comment]
> This is a feature that you need to enable in your configuration as it is still considered experimental. I just put together a page explaining the feature and how to enable it: https://www.navidrome.org/docs/usage/sharing/

Have you considered also a per-track permalink, so it can be shared with other logged users?
https://github.com/navidrome/navidrome/issues/1650

A link like https://navi.drome/app/#/track/$trackid

**Reply by deluan** [reply]
This may be implemented in the new UI that we are working on.

**Comment by ztx-lyghters** [comment]
It would be nice to have a standard for "direct links" to certain resources/objects, like artists, albums or public playlists.

Not just a simple URL, but something "universal" (even if specific to Navidrome), not tied to a domain name (but containing the information about it), so any Navidrome/Subsonic client could intercept it, parse it and open the page of the resource/object in question. The way I see it, that link could have a domain name information and ID of the object in question (maybe also some additional information).

It would make sharing between the users of the same server so much easier, more functional and client-independent, without the need to create actual "shares" with very limited functionality/presentation every time. And yes, just to clarify, this won't be a "public access" kind of share, it would be specifically for those, who have access to the instance.

Would be even better for something like this to be implemented in the OpenSubsonic protocol as well.
